com.hma.qlog.yml

```diff
@@ -0,0 +1,43 @@
+app-id: com.hma.qlog
```

Comment:
```suggestion
app-id: io.github.7h0ma5.QLog
```

Similar change is needed for desktop, appdata, etc.

Reply:
ok

Reply:
In the name 7h0ma5 of the application is not valid I imagine that 7h0ma5 refers to Thomas I will have to give him that name

Reply:
error: 'io.github.7h0ma5.QLog' is not a valid application name: Name segment can't start with 7